### PR TITLE
[544435] regenerated xtend

### PR DESCRIPTION
**Pull request by cdietrich**
[544435] regenerated xtend
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=544435

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=544435

File: org.eclipse.xtend.core/src/org/eclipse/xtend/core/serializer/XtendSyntacticSequencer.java [hunk 1]
```diff
@@ -82,4 +82,11 @@ protected void emit_XBlockExpression_SemicolonKeyword_2_1_q(EObject semanticObje
 		} else
 			acceptNodes(transition, nodes);
 	}
+	
+	@Override
+	protected String getArrayBracketsToken(EObject semanticObject, RuleCall ruleCall, INode node) {
+		if (node != null)
+			return getTokenText(node);
+		return "[]";
```
Comment (kthoms) [inline org.eclipse.xtend.core/src/org/eclipse/xtend/core/serializer/XtendSyntacticSequencer.java]: Why does the above have a space and this not?

Comment (cdietrich) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/serializer/XtendSyntacticSequencer.java]: See the bug

Comment (cdietrich) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/serializer/XtendSyntacticSequencer.java]: This is a fix for the introduced space in superclass

**Pull request merged**